src/test/java/wooteco/subway/acceptance/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,194 @@
+package wooteco.subway.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import wooteco.subway.domain.Section;
+import wooteco.subway.domain.Station;
+import wooteco.subway.dto.StationResponse;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("구간 관련 기능")
+public class SectionAcceptanceTest extends AcceptanceTest {
+
+    @BeforeEach
+    void setUpData() {
+        createStation("강남역");
+        createStation("역삼역");
+        createStation("선릉역");
+        createStation("잠실역");
+        createLine("2호선", "bg-red-600", 1L, 2L, 10);
+    }
+
+    @Test
+    @DisplayName("하행에 정상적으로 구간을 연결하는 경우를 테스트한다")
+    void createSectionTest() {
+        ExtractableResponse<Response> response = createSection(2L, 3L, 5);
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+    }
+
+    @Test
+    @DisplayName("이미 등록된 상행과 하행을 연결하는 경우를 테스트한다.")
```

Comment:
```suggestion
    @DisplayName("이미 등록된 상행과 하행을 역으로 가지는 구간을 추가할 경우 400 에러를 반환한다.")
```

제 제안과 라쿤이 기존에 작성해주신 내용은 어떤 차이가 있을까요?

아래에 몇몇 DisplayName에는 상세하게 작성해주셨지만 부족한 부분도 있기에 리뷰 남겨요 😀

Reply:
DisplayName에서 요구사항을 만족하는지(적절한 응답 코드를 반환하는지)를 확인할 수 있겠네요😀

DisplayName에 어느 정도까지 자세하게 적어야 하는지에 대한 기준이 모호한 거 같아요😅 적절한 응답 코드까지 표현해 

주어야 하는지, 아니면 `연결하는 경우 예외가 발생한다` 라고 써주어도 될까요?

Reply:
일단은 예외가 발생한다 라고 수정하였습니다!! 😀

Reply:
라쿤이 결정하면 되어요. 

코드를 읽는 사람이 400 에러에 대해서 이해하고 있는가, 혹은 이해해야 하는가도 고민해보면 라쿤의 방향성도 좋다 생각합니다! 우테코 기간 동안은 여러가지를 시도해보고 학습하며 라쿤만의 기준을 세워가면 좋겠어요. 고민하시고 적절하게 반영해주셨네요! 굳굳